.lintstagedrc

```diff
@@ -1,7 +1,8 @@
 {
   "linters": {
     "**/*{.ts,.tsx,.js,.jsx,.css,.json,.md}": [
-      "prettier --write"
+      "prettier --write",
+      "git add"
```

Comment:
Will this prevent partial staging/commit of a file?

Reply:
The file was by definition already staged, this makes sure that the changes are included in the commit.

Reply:
You can do partial staging with e.g. `git add -p` or `git gui`, where you stage chunks/lines. I think running `git add` automatically will override any such partial staging.

Reply:
C.f. https://github.com/okonet/lint-staged/issues/62

Reply:
I'm not stepping into that landmine.  This change only corrects on omission that brings it in line with the other two linters in this file.

Reply:
Sure, I just wonder if there are some way for me to disable these actions? I use partial staging a lot, so I'd rather learn which commands to run!

Reply:
I don't know how to handle that case

Reply:
I'll try to open up another PR for it.

Reply:
Great!